recipes/fmt/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.1.2)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.15)
```

Actually it's probably a bug in `CMakeToolchain`, https://cmake.org/cmake/help/latest/policy/CMP0091.html should be set to NEW, because even if the toolchain checks that CMake is not < 3.15, the CMakeLists may be lower than 3.15 , so implicitly CMP0091 policy is OLD, therefore ignoring `CMAKE_MSVC_RUNTIME_LIBRARY`.